Cargo.lock

```diff
@@ -2517,6 +2594,12 @@ dependencies = [
  "ws",
 ]
 
+[[package]]
+name = "keccak"
```

Comment:
hmm, another keccak library?

Reply:
https://github.com/AgeManning/enr/pull/7

Reply:
We are on v1.x and the upgrade to 2 is a fair amount of work. Afaik v2 does not bring perf improvements. Not sure using it here is a good idea?

Reply:
I'll take it upon myself to upgrade the code to tiny-keccak v2 after this.